onnxruntime/test/perftest/README.md

```diff
@@ -1,25 +1,53 @@
 # ONNXRuntime Performance Test
 
-onnxruntime_perf_test [options...] model_path result_file
+This tool provides the time for the ONNX Runtime with the specific execution provider to run a inference for a given model using the given test date. This tool can provide a reliable measurement for the inference latency usign ONNX Runtime on the HW platform. The options to use with the tool are listed below:
+
+`onnxruntime_perf_test [options...] model_path result_file`
+
 Options:
-        -m [test_mode]: Specifies the test mode. Value coulde be 'duration' or 'times'.
-                Provide 'duration' to run the test for a fix duration, and 'times' to repeated for a certain times. Default:'duration'.
-        -e [cpu|cuda|mkldnn|tensorrt]: Specifies the provider 'cpu','cuda','mkldnn' or 'tensorrt'. Default:'cpu'.
-        -r [repeated_times]: Specifies the repeated times if running in 'times' test mode.Default:1000.
-        -t [seconds_to_run]: Specifies the seconds to run for 'duration' mode. Default:600.
-        -p [profile_file]: Specifies the profile name to enable profiling and dump the profile data to the file.
-        -s: Show statistics result, like P75, P90.
-        -v: Show verbose information.
-        -x: Use parallel executor, default (without -x): sequential executor.
-        -h: help
+
+	-m [test_mode]: Specifies the test mode. Value coulde be 'duration' or 'times'.
```

Comment:
perf_test [options...] model_path result_file
Options:
        -m [test_mode]: Specifies the test mode. Value could be 'duration' or 'times'.
                Provide 'duration' to run the test for a fix duration, and 'times' to repeated for a certain times.
        -M: Disable memory pattern.
        -A: Disable memory arena
        -c [parallel runs]: Specifies the (max) number of runs to invoke simultaneously. Default:1.
        -e [cpu|cuda|mkldnn|tensorrt|ngraph|openvino|nuphar]: Specifies the provider 'cpu','cuda','mkldnn','tensorrt', 'ngraph', 'openvino' or 'nuphar'. Default:'cpu'.
        -r [repeated_times]: Specifies the repeated times if running in 'times' test mode.Default:1000.
        -t [seconds_to_run]: Specifies the seconds to run for 'duration' mode. Default:600.
        -p [profile_file]: Specifies the profile name to enable profiling and dump the profile data to the file.
        -s: Show statistics result, like P75, P90.
        -v: Show verbose information.
        -x [intra_op_num_threads]: Sets the number of threads used to parallelize the execution within nodes, A value of 0 means ORT will pick a default. Must >=0.
        -y [inter_op_num_threads]: Sets the number of threads used to parallelize the execution of the graph (across nodes), A value of 0 means ORT will pick a default. Must >=0.
        -P: Use parallel executor instead of sequential executor.
        -o [optimization level]: Default is 1. Valid values are 0 (disable), 1 (basic), 2 (extended), 99 (all).
                Please see onnxruntime_c_api.h (enum GraphOptimizationLevel) for the full list of all optimization levels.
        -h: help

Reply:
The original doc is outdated.

Reply:
Updated.